_releases/2022-11-08-october-2022.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+category: releases
+title: October 2022 Release Notes
+filename-custom: october-2022
+last_modified_at: November 8, 2022
+sidenav: about_us
+release-year: "2022"
+description: ""
+---
+## Improvements
+
+* **ElasticSearch Upgrade**: We’ve continued testing and planning for the ElasticSearch upgrade.
+* **Search Engine Results Page (SERP) Redesign Investigations**: We've explored a few options to use more modern frontend technology to power our SERP.
+* **Rails 7 Upgrade**: We've started upgrading components of our search-gov app in preparation for Rails 7.
```

Comment:
Please remove the version number. The second mention could become "for the next Rails version/"

Reply:
Updated!